solr/core/src/java/org/apache/solr/cli/RunExampleTool.java

```diff
@@ -869,6 +857,7 @@ protected boolean isValidConfig(File configsetsDir, String config) {
   }
 
   protected Map<String, Object> getNodeStatus(String solrUrl, int maxWaitSecs) throws Exception {
+    solrUrl += "/solr";
```

Comment:
bummer!  But we are in an awkward place where a solrUrl can end with /solr or with /api ;-)  So maybe this pattern becomes more common in varous tools...  Actually, are you sure you need this?  Ithought the status tool would deal with the solrURL.....